website/versioned_docs/version-0.11/plugins/ax_sweeper.md

```diff
@@ -0,0 +1,55 @@
+---
```

Comment:
Since this plugin does not work in 0.11, do not document it here.
(Unless you add a followup diff that backports it to 0.11).

To view the latest doc version, append "/next" to the url after /docs.

Reply:
Hmm there is not `versioned_sidebar` for master. Should I make one?